linera-views/src/backends/dynamo_db.rs

```diff
@@ -91,11 +96,17 @@ async fn get_config_internal(
     }
 }
 
+/// DynamoDB forbids the iteration over the partition keys.
+/// Therefore we use a special partition key named `[1]` for storing
+/// the root keys. For normal root keys, we simply put a `[0]` in
+/// front therefore no intersection is possible.
+const PARTITION_KEY_ROOT_KEY: &[u8] = &[1];
+
 /// The attribute name of the partition key.
 const PARTITION_ATTRIBUTE: &str = "item_partition";
 
 /// A root key being used for testing existence of tables
```

Comment:
Is this a "key" or a "key prefix"?

Reply:
It is a key.
The partition key coming from root keys start with 0, the partition key storing the root keys is exactly `[1]`.